### DISCUSSION TITLE
请问Mac如何使用Maa？

### DISCUSSION THREAD

**Discussion by Gloaming02** [MaaAssistantArknights/MaaAssistantArknights#3115] [Q&A] [answered]
我下载了夜神模拟器Mac版本，但是Maa没有MAC版本，难道我要使用Linux版本的吗。我没试过，是可以的吗？

**Comment by ABA2396** [comment, marked as answer]
480版本用了onnx文字识别，mac还没支持，你可以尝试473版本，那个是有mac版本的
![image](https://user-images.githubusercontent.com/99072975/208366373-57d397a1-df00-4d07-8f6f-736938780dc2.png)
